### ISSUE TITLE
Progress bar with warning state and visible value fails colour contrast

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1349]
Code to replicate: 

```
{{
        html.progress({
            class: 'progress-bar--warning',
            value: 70,
            value_visible: true
        })
}}

```

![image](https://user-images.githubusercontent.com/756393/103542159-6ca7e180-4e94-11eb-8f0c-dcfe775c0286.png)
